hieradata/role/rucio.yaml

```diff
@@ -1,12 +1,5 @@
 ---
 classes:
   - "profile::core::common"
-  - "profile::core::debugutils"
+  - "profile::core::nfsclient"
   - "profile::core::rucio"
-  - "python"
-
-python::version: "python36"
-python::pip: "present"
-python::dev: "present"
-python::virtualenv: "present"
-profile::core::common::manage_firewall: false
```

Comment:
I would like all new or heavily updated rules for here forward to enable iptables/nft rules.  I have already started doing this. See the puppetdb role for a good example of this: https://github.com/lsst-it/lsst-control/blob/3b2a02929b41363945b8ce935255b0b15d104d52/hieradata/role/puppetdb.yaml#L28-L48